### PR TITLE
Pbw 6756

### PR DESCRIPTION
**Pull request by gramcha**

### CODE REVIEW

File: skin.json [hunk 1]
```diff
@@ -155,7 +156,8 @@
   },
   "controlBar": {
     "volumeControl": {
-      "color": ""
+      "color": "",
+      "inactiveColor": ""
```
Comment (pilievOoyala) [inline skin.json]: Do we need this?  It looks like something left over  from a previous change/

File: skin-schema.json [hunk 2]
```diff
@@ -507,6 +515,14 @@
               "iOS SDK": "Unsupported",
               "Android SDK": "v4.21.0+"
             }
+          },
+          "inactiveColor" : {
```
Comment (pilievOoyala) [inline skin-schema.json]: Please make the naming scheme consistent.  So this should be inactiveAccentColor

Comment (pilievOoyala) [inline skin-schema.json]: What is the difference between these two colors?

File: skin.json [hunk 3]
```diff
@@ -9,7 +9,8 @@
                   "scalingPercentage": 20
     },
     "loadingImage": {"imageResource": {"url": "//player.ooyala.com/static/v4/candidate/latest/skin-plugin/assets/images/loader_svg.svg" } },
-    "accentColor":"#448aff"
+    "accentColor":"#448aff",
+    "inactiveAccentColor":"rgba(175,175,175,0.5)"
```
Comment (pilievOoyala) [inline skin.json]: can you make the color number consistent instead of having two different styles here?

File: skin-schema.json (hunk 2 above)
Comment (pilievOoyala) [reply skin-schema.json]: Nevermind, the diff made it look like they were specified next to each other. This makes more sense if I look at the file as a whole.

**Comment by gramcha** [comment]
The different solution provided. This PR not required

**Pull request closed without merging**